docs/src/pages/HomepageFeatures.js

```diff
@@ -9,20 +9,20 @@ const FeatureList = [
     Svg: require("../../static/img/icons8-windows-11.svg").default,
     description: (
       <>
-        Uses{" "}
+        Supports{" "}
         <Link to="https://github.com/rdbende/Sun-Valley-TTk-Theme">
           {" "}
-          @rdbende's Sun Valley TTk Theme
+          Fluent Design Themeing
         </Link>
-        . Feels modern!
+        and Mica. Feels modern!
```

Comment:
Nit:
missing space before 'and'
Themeing -> Theming

Reply:
I'll try fixing these now.